### ISSUE TITLE
Feature Request: Update it to use Npcap instead of WinPcap on Windows

### ISSUE THREAD

**Issue by pfjarschel** [phaethon/scapy#173]
Hi there! First of all, nice work with this fork! Makes everything so much easier! However...

The latest main scapy version (2.3.3) has updated the api to use npcap instead of winpcap, which is a lot better, since winpcap doesn't suport wifi monitor mode, for example. It would be really nice if scapy3 could also use npcap!

I know that this is probably not a priority, but... :)

Oh, and I'm sorry if this is already known, or if there is a way to make it work with npcap!

**Comment by phaethon** [comment]
Yes, it would be a good idea to have npcap supported. No specific due date planned yet. Any help with the code will be appreciated :)

**Comment by gpotter2** [comment]
@pfjarschel 
Hi, for the record, the latest main scapy version does now supports Python 3. If you want to use Python 3 with npcap, you can have a look at it :)

https://github.com/secdev/scapy

Good day
